recipes/python-symengine/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export CC=clang
+    export CXX=clang++
+
+    export MACOSX_VERSION_MIN=10.9
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+    export LINKFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11 -L${LIBRARY_PATH}"
+
+    # This enables Xcode support for C++11, that is needed for SymEngine.
+    export MACOSX_DEPLOYMENT_TARGET=10.9
+fi
```

Comment:
Is, `export MACOSX_DEPLOYMENT_TARGET=10.9` added by default when you do `source activate "${CONDA_DEFAULT_ENV}"`?